sdk/storage/Azure.Storage.DataMovement/src/TransferCheckpointStoreOptions.cs

```diff
@@ -10,9 +10,24 @@ namespace Azure.Storage.DataMovement
     public class TransferCheckpointStoreOptions
     {
         /// <summary>
-        /// The local folder where the checkpoint information will be stored.
+        /// Whether checkpointing should be enabled or not.
+        /// </summary>
+        internal bool Enabled { get; private set; }
+
+        /// <summary>
+        /// The folder where the checkpoint information will be stored.
         /// </summary>
-        public string CheckpointerPath { get; private set; }
+        internal string CheckpointerPath { get; private set; }
```

Comment:
Make sure you update the changelog for this breaking change from public to internal